Makefile

```
@@ -273,7 +273,8 @@ kind-down: $(KIND)
 	rm -rf dev/local-backupbuckets
 
 # workaround for https://github.com/GoogleContainerTools/skaffold/issues/6416
-export SKAFFOLD_LABEL := skaffold.dev/run-id=gardener-local
+$(SKAFFOLD): export SKAFFOLD_LABEL := skaffold.dev/run-id=gardener-local
```

```suggestion
gardener-up gardener-down: export SKAFFOLD_LABEL = skaffold.dev/run-id=gardener-local
```

On my macOS default make `SKAFFOLD_LABEL` is not set on `make gardener-up` when the export is assigned to a dependent target.

---

However, the issue mentioned there is closed and it seems to work without exporting the env anyhow. Maybe we don't need the workaround anymore.

---

Well, I think https://github.com/GoogleContainerTools/skaffold/pull/7011 is the fix we need for etcd. However, not setting `SKAFFOLD_LABEL` will lead to a complete rollout of all gardener components every time you run `make gardener-up`. 
I would vote for keeping it in this case.

I will push some updates tomorrow for upgrading skaffold and also applying your suggestion.

---

ah ok, this makes sense 😄 . I did not know that otherwise skaffold changes labels on every run
Previously, I just noticed that the env is not set. However, `make gardener-up` is still working and the PR from the comment was merged. 
You could remove the `# workaround for https://github.com/GoogleContainerTools/skaffold/issues/6416` comment then. The env won't be a workaround anymore.